openapi_python_client/parser/properties/model_property.py

```diff
@@ -32,6 +32,7 @@ class ModelProperty(PropertyProtocol):
     relative_imports: set[str] | None
     lazy_imports: set[str] | None
     additional_properties: Property | None
+    ref_path: ReferencePath | None = None
```

Comment:
See this comment: https://github.com/openapi-generators/openapi-python-client/pull/1154/files#diff-91d61e88486ae43dc0579d44f7964667c53749e7091907d6a064d80b57a4b26dR145